src/alire/alire-os_lib-subprocess.ads

```diff
@@ -2,17 +2,25 @@ with Alire.Utils;
 
 package Alire.OS_Lib.Subprocess is
 
-   --  Separate from Alire.OS_Lib because not preelaborable
+   --  Separate from Alire.OS_Lib because not preelaborable.
```

Comment:
We usually don't add dots (`.`) for single line comments.

Reply:
Understood